### ISSUE TITLE
retry_wait_seconds does not work

### ISSUE THREAD

**Issue by ahansson89** [nick-invision/retry#5]
```      
    - name: Create AWS Secrets
        if: steps.semantic.outputs.new-release-published == 'true'
        uses: nick-invision/retry@v1
        with:
          timeout_minutes: 1
          max_attempts: 60
          retry_wait_seconds: 5
          command: aws secretsmanager create-secret --name stripeSecretKey --secret-string ${STRIPE_KEY}```

Using it this way, but it retry's every second and does not wait

**Comment by nick-invision** [comment]
@ahansson89 Can you try again? I just fixed an issue in v1.0.4 (v1 alias was moved to v1.0.4 also so it should still work) where retry_wait_seconds was ignored if the process exits with a non-zero code.  I believe this is what you have encountered.

**Comment by ahansson89** [comment]
Looks to be working perfectly now. Thanks! Great plugin!
